modules/500-openvpn/openapi/doc-ru-config-values.yaml

```
@@ -131,6 +131,12 @@ properties:
       Структура, аналогичная  `spec.tolerations` в Kubernetes Pod.
 
       Если ничего не указано или указано `false` — будет [использоваться автоматика](https://deckhouse.ru/documentation/v1/#выделение-узлов-под-определенный-вид-нагрузки).
+  pmacctEnabled:
+    description: |
+      Включить логирование IP соединений в виде json-логов для клиентских openvpn подключений.
```

Mind the foreign-word-based-words, there must be dash.
```suggestion
      Включить логирование IP-соединений в виде json-логов для клиентских openvpn-подключений.
```